### PR TITLE
Sort events by date before rendering them, add tests to verify expected behavior.  

### LINKED ISSUE #13: Sort Events Chronologically
When adding issues, they seem to be displayed (both in reminders and via the `list` command) in the order the events were added.  It would be nice (not sure if better would be the right word) to see them sorted chronologically.

Example of current behavior:
`@callie list`

Your events:
1: Auntie Em's Birthday on 03/12/2020
2: Trip to Paris on 11/13/2019
3: Going away party for Ed from Accounting on 09/13/2019

What (I think) would look more intuitive:
`@callie list`

Your events:
1: Going away party for Ed from Accounting on 09/13/2019
2: Trip to Paris on 11/13/2019
3: Auntie Em's Birthday on 03/12/2020

### PR DESCRIPTION
**Pull request by tredontho**
I've never done any actual javascript development before.  I wasn't able to test this change extensively, I wrote a pretty simple/dumb test that makes me feel somewhat confident, but please let me know if there's a better way to test this out or if you have any other feedback.

Resolves #13

### CODE REVIEW

File: src/__test__/countdowns.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import * as R from 'ramda';
+import '@babel/polyfill';
+
+import { notify } from '../services/message.service';
+import findCountdowns from '../queries/findCountdowns';
+import listCountdowns from '../lib/listCountdowns';
+import moment from 'moment';
+
+const TEST_EVENTS = [
+  {event: "Two", date: "2019-02-02"},
```
Comment (brianpgerson) [inline src/__test__/countdowns.test.js]: nit - preference is for `'`

File: src/__test__/countdowns.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  {event: "Four", date: "2019-04-04"},
+  {event: "Three", date: "2019-03-03"},
+];
+
+const EXPECTED = `Your events:\n 1: One _on_ ${moment("2019-01-01").calendar()}\n2: Two _on_ ${moment("2019-02-02").calendar()}\n3: Three _on_ ${moment("2019-03-03").calendar()}\n4: Four _on_ ${moment("2019-04-04").calendar()}`
+
+jest.mock('../services/message.service')
+notify.mockImplementation(jest.fn(message => context => {return message;}))
+jest.mock('../queries/findCountdowns');
+findCountdowns.mockResolvedValue(TEST_EVENTS);
```
Comment (brianpgerson) [inline src/__test__/countdowns.test.js:21]: 💯

File: src/lib/listCountdowns.js [hunk 3]
```diff
@@ -20,6 +21,10 @@ const formatAndNotify = configuration => countdowns => R.pipe(
   message => notify(message)(configuration)
 )(countdowns)
 
+const sortEventsByDate = (events) => {
```
Comment (brianpgerson) [inline src/lib/listCountdowns.js]: you can actually just do

```
const sortEventsByDate = (events) =>  R.sort(
    (e0, e1) => moment(e0.date).isBefore(e1.date) ? -1 : (moment(e0.date).isSame(e1.date) ? 0 : 1), 
    events
)
```

**Review by brianpgerson — COMMENTED** [review]
this looks great! you are, i believe, the first contributor to callie, so thanks for doing this. 

i'll approve and merge when i'm home from work and can briefly run it to verify locally.  

thanks again!

File: src/__test__/countdowns.test.js (hunk 1 above)
Comment (tredontho) [reply src/__test__/countdowns.test.js]: Easy change, will do.

File: src/lib/listCountdowns.js (hunk 3 above)
Comment (tredontho) [reply src/lib/listCountdowns.js]: D'oh!  I realized when I walked out this morning that I had missed this.  Fixing it ASAP

**Comment by tredontho** [comment]
This only handles the `@callie list` command, it doesn't handle sorting for the reminders.  I'm not sure how to tackle that yet, maybe it will be a separate PR unless you think it'll be easy to squeeze in here.

**Review by brianpgerson — APPROVED** [review]
🙇

**Comment by brianpgerson** [comment]
your change has been deployed. thanks for your contribution!

**Pull request merged**